invenio_communities/assets/semantic-ui/js/invenio_communities/members/components/bulk_actions/SearchResultsBulkActions.js

```diff
@@ -66,8 +69,8 @@ export class SearchResultsBulkActions extends Component {
             options={dropdownOptions}
             item
             selection
-            value={action}
-            defaultValue={undefined}
+            value={null}
+            selectOnBlur={false}
```

Comment:
I think also `selectOnNavigation={false}` is needed here to avoid that the option is selected, and the dropdown closing, when navigating with a screenreader.